### ISSUE TITLE
Bug: no-multiple-empty-lines is detected in strings

### ISSUE THREAD

**Issue by AhmedAtef07** [vsiakka/gherkin-lint#143]
`no-multiple-empty-lines` should not be triggered in strings.
<img width="551" alt="screen shot 2019-02-12 at 13 47 22" src="https://user-images.githubusercontent.com/4195363/52639708-c253f900-2ecc-11e9-99b6-13b298662db1.png">

**Comment by xgbuils** [comment]
Hi @AhmedAtef07,

What is the use case of forbidding multilines in strings?

**Comment by AhmedAtef07** [comment]
@xgbuils I have it in a screenshot in the issue description.

But here is it again written here:
```gherkin 
Scenario: Empty lines should be rendered as empty lines 
  When user sends a message:
  """
  1
  
  2
 

  3
  """

**Comment by xgbuils** [comment]
Hi, @AhmedAtef07 

I don't know too much about DocString. My question was about why is applying the rule over DocStrings useful. I understand that a new lines between steps, scenarios, etc doesn't have meaning. It's just a stylistic thing. However a DocString with one new line or two new lines could have different meanings. I'm not sure.

**Comment by xgbuils** [comment]
Hi @AhmedAtef07,

Sorry. I thought you say that `no-multiple-empty-lines` SHOULD be triggered in strings. But you are saying the opposite.

**Comment by vsiakka** [comment]
I think that the intent of the rule is to find multiple new lines between steps. Because in @AhmedAtef07 's example the string is part of a docstring, so it's part of a single step definition and you should be able to put anything you want in your step, the rule shouldn't trigger. 

I'm not sure about how to fix this yet, I'll need to look at the tree that gherkin generates closer and see if i can extract the starting and ending line of docstrings and exclude any line in between from the errors.
